Makefile

```diff
@@ -95,6 +95,14 @@ run-g: build
 run-ps: build
 	./bin/${APP_NAME} --providers
 
+# normalize all go files before push to git repo
+normalize:
+	go mod tidy
+	go test -test.timeout=10s ./...
+	find ./ -name "*.go" | xargs gofmt -w -l
+	find ./ -name "*.go" | goimports -w l
+	golint -set_exit_status ./...
```

Comment:
why no golangci-lint enable? where ci action use to check

Reply:
use it at local before push git